docs/jax-101/index.rst

```diff
@@ -0,0 +1,14 @@
+Tutorial: JAX 101 (Work in Progress)
+====================================
+
+This is a tutorial developed by engineers and researchers at DeepMind_.
+It is a work in progress; more content will be added soon!
+
+.. toctree::
+  :maxdepth: 2
```

Comment:
nit: I think once we have more guides, this will be easier to read with :maxdepth: 1 (i.e. only show the tutorial titles and not the sections)

Reply:
Here because of the index the top tile is "JAX 101", and the second level is the tutorial titles, so I think it's what we want.